ajax/libs/howler/package.json

```diff
@@ -1,16 +1,20 @@
 {
   "name": "howler",
   "filename": "howler.min.js",
-  "version": "1.1.29",
+  "version": "2.0.0",
   "description": "Javascript audio library for the modern web.",
-  "homepage": "http://howlerjs.com/",
+  "homepage": "https://howlerjs.com/",
   "keywords": [
-    "audio"
+    "howler",
+    "howler.js",
+    "audio",
+    "sound",
+    "webaudio"
   ],
   "autoupdate": {
     "source": "git",
     "target": "git://github.com/goldfire/howler.js.git",
-    "basePath": "",
+    "basePath": "dist",
     "files": [
```

Comment:
What about using a single pattern `**/*` in filemap to grab all the files under `dist`?